src/ce/crt0.src

```diff
@@ -143,21 +141,67 @@ assert mpTmr2Counter shr 8 = mpTmrCtrl shr 8
 	ld	l, mpTmrCtrl and $FF
 	set	bTmr2Enable, (hl)
 
+	section	.text
+	private	___rerun_handler
+	private	___rerun_handler.callback
+	private	___rerun_handler.data
+	private	___rerun_handler.retval
+___rerun_handler:
+	ld	hl,(_exit.sp)
+	ld	de,0
+.data_size := $ - 3
+	add	hl,de
+	ld	(_exit.sp),hl		; move exit.sp to real ret
+	ld	hl,0
+.retval := $ - 3
+	push	hl
+	ld	hl,0
+.data := $ - 3
+	push	hl
+	call	0
+.callback := $ - 3
+	pop	de, de
+	ret
```

Comment:
There's no reason to fix sp, main arguments are certainly never manually removed.
```suggestion
	ex	(sp),hl
	ld	de,0
.data := $ - 3
	push	de,hl
	jp	0
.callback := $ - 3
```

Reply:
fixed and updated